### ISSUE TITLE
BUG on upgrade to 0.28.0 RC 2

### ISSUE THREAD

**Issue by jonatasfreitasv** [metabase/metabase#6852]
```
Creating metabase ... done
Attaching to metabase
metabase    | Picked up JAVA_TOOL_OPTIONS: -Xmx14000m -Xms2000m
metabase    | 02-02 20:29:46 INFO metabase.util :: Loading Metabase...
metabase    | 02-02 20:29:50 INFO util.encryption :: DB details encryption is DISABLED for this Metabase instance. 🔓 
metabase    | See http://www.metabase.com/docs/latest/operations-guide/start.html#encrypting-your-database-connection-details-at-rest for more information.
metabase    | 02-02 20:29:55 INFO metabase.core :: Starting Metabase in STANDALONE mode
metabase    | 02-02 20:29:55 INFO metabase.core :: Launching Embedded Jetty Webserver with config:
metabase    |  {:port 3000, :host "0.0.0.0"}
metabase    | 
metabase    | 02-02 20:29:55 INFO metabase.core :: Starting Metabase version v0.28.0-RC2 (22be7fd release-0.28.0) ...
metabase    | 02-02 20:29:55 INFO metabase.core :: System timezone is 'GMT' ...
metabase    | 02-02 20:29:56 INFO metabase.core :: Setting up and migrating Metabase DB. Please sit tight, this may take a minute...
metabase    | 02-02 20:29:56 INFO metabase.db :: Verifying postgres Database Connection ...
metabase    | 02-02 20:29:56 INFO metabase.db :: Verify Database Connection ...  ✅
metabase    | 02-02 20:29:56 INFO metabase.db :: Running Database Migrations...
metabase    | 02-02 20:29:56 INFO metabase.db :: Setting up Liquibase...
metabase    | 02-02 20:29:56 INFO metabase.db :: Liquibase is ready.
metabase    | 02-02 20:29:56 INFO metabase.db :: Checking if Database has unrun migrations...
metabase    | 02-02 20:29:58 INFO metabase.db :: Database has unrun migrations. Waiting for migration lock to be cleared...
metabase    | 02-02 20:29:58 INFO metabase.db :: Migration lock is cleared. Running migrations...
metabase    | 02-02 20:30:00 INFO metabase.db :: Database Migrations Current ...  ✅
metabase    | com.mchange.v2.cfg.DelayedLogItem [ level -> FINE, text -> "The configuration file for resource identifier 'hocon:/reference,/application,/c3p0,/' could not be found. Skipping.", exception -> null]
metabase    | 02-02 20:30:01 INFO db.migrations :: Running all necessary data migrations, this may take a minute.
metabase    | 02-02 20:30:01 INFO db.migrations :: Running data migration 'migrate-humanization-setting'...
metabase    | 02-02 20:30:01 INFO models.humanization :: Now using simple table name humanization.
metabase    | 02-02 20:30:01 INFO models.humanization :: Updating display name for Field 'DLR': 'DLR' -> 'Dlr'
metabase    | org.postgresql.util.PSQLException: ERROR: column "id" does not exist
metabase    |   Position: 29
metabase    | 	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2477)
metabase    | 	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2190)
metabase    | 	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:300)
metabase    | 	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:428)
metabase    | 	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:354)
metabase    | 	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:169)
metabase    | 	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:136)
metabase    | 	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:105)
metabase    | 	at clojure.java.jdbc$db_do_execute_prepared_statement$fn__10897.invoke(jdbc.clj:952)
metabase    | 	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:741)
metabase    | 	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:711)
metabase    | 	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:724)
metabase    | 	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:711)
metabase    | 	at clojure.java.jdbc$db_do_execute_prepared_statement.invokeStatic(jdbc.clj:951)
metabase    | 	at clojure.java.jdbc$db_do_execute_prepared_statement.invoke(jdbc.clj:945)
metabase    | 	at clojure.java.jdbc$db_do_prepared.invokeStatic(jdbc.clj:983)
metabase    | 	at clojure.java.jdbc$db_do_prepared.invoke(jdbc.clj:963)
metabase    | 	at clojure.java.jdbc$execute_BANG_$execute_helper__10968.invoke(jdbc.clj:1364)
metabase    | 	at clojure.java.jdbc$execute_BANG_.invokeStatic(jdbc.clj:1368)
metabase    | 	at clojure.java.jdbc$execute_BANG_.invoke(jdbc.clj:1337)
metabase    | 	at toucan.db$execute_BANG_.invokeStatic(db.clj:416)
metabase    | 	at toucan.db$execute_BANG_.doInvoke(db.clj:411)
metabase    | 	at clojure.lang.RestFn.invoke(RestFn.java:410)
metabase    | 	at toucan.db$simple_delete_BANG_.invokeStatic(db.clj:729)
metabase    | 	at toucan.db$simple_delete_BANG_.invoke(db.clj:714)
metabase    | 	at toucan.db$simple_delete_BANG_.invokeStatic(db.clj:732)
metabase    | 	at toucan.db$simple_delete_BANG_.doInvoke(db.clj:714)
metabase    | 	at clojure.lang.RestFn.invoke(RestFn.java:445)
metabase    | 	at toucan.db$delete_BANG_.invokeStatic(db.clj:749)
metabase    | 	at toucan.db$delete_BANG_.doInvoke(db.clj:734)
metabase    | 	at clojure.lang.RestFn.invoke(RestFn.java:439)
metabase    | 	at metabase.db.migrations$migrate_humanization_setting.invokeStatic(migrations.clj:362)
metabase    | 	at metabase.db.migrations$migrate_humanization_setting.invoke(migrations.clj:350)
metabase    | 	at metabase.db.migrations$run_migration_if_needed_BANG_.invokeStatic(migrations.clj:50)
metabase    | 	at metabase.db.migrations$run_migration_if_needed_BANG_.invoke(migrations.clj:41)
metabase    | 	at metabase.db.migrations$run_all_BANG_.invokeStatic(migrations.clj:70)
metabase    | 	at metabase.db.migrations$run_all_BANG_.invoke(migrations.clj:64)
metabase    | 	at clojure.lang.Var.invoke(Var.java:375)
metabase    | 	at metabase.db$run_data_migrations_BANG_.invokeStatic(db.clj:390)
metabase    | 	at metabase.db$run_data_migrations_BANG_.invoke(db.clj:385)
metabase    | 	at metabase.db$setup_db_BANG_.invokeStatic(db.clj:401)
metabase    | 	at metabase.db$setup_db_BANG_.doInvoke(db.clj:392)
metabase    | 	at clojure.lang.RestFn.invoke(RestFn.java:421)
metabase    | 	at metabase.core$init_BANG_.invokeStatic(core.clj:141)
metabase    | 	at metabase.core$init_BANG_.invoke(core.clj:120)
metabase    | 	at metabase.core$start_normally.invokeStatic(core.clj:249)
metabase    | 	at metabase.core$start_normally.invoke(core.clj:242)
metabase    | 	at metabase.core$_main.invokeStatic(core.clj:347)
metabase    | 	at metabase.core$_main.doInvoke(core.clj:342)
metabase    | 	at clojure.lang.RestFn.invoke(RestFn.java:397)
metabase    | 	at clojure.lang.AFn.applyToHelper(AFn.java:152)
metabase    | 	at clojure.lang.RestFn.applyTo(RestFn.java:132)
metabase    | 	at metabase.core.main(Unknown Source)
metabase    | 02-02 20:30:01 ERROR metabase.core :: Metabase Initialization FAILED:  ERROR: column "id" does not exist
metabase    |   Position: 29
metabase    | 02-02 20:30:01 INFO metabase.core :: Metabase Shutting Down ...
metabase    | 02-02 20:30:01 INFO metabase.core :: Metabase Shutdown COMPLETE
metabase exited with code 1
```

**Comment by camsaul** [comment]
Fixed by #6854

**Comment by camilonova** [comment]
I'm having this same issue after upgrading to 0.28.5
